public/src/store.jsx

```diff
@@ -25,6 +26,8 @@ const rootReducer = (state, action) => {
 
 const middleware = [immutable(), createLogger(), promise(), thunk];
 
-const store = createStore(rootReducer, compose(applyMiddleware(...middleware)));
+const store = createStore(rootReducer, compose(applyMiddleware(...middleware), autoRehydrate()));
+
+persistStore(store);
```

Comment:
I think it's whenever the server is restarted, and I think auto-rehydrate keeps the store from getting stale over time.